packages/cuba-rest-js/src/event_emitter.ts

```diff
@@ -0,0 +1,58 @@
+interface IKeyFuncVal {
+    [k: string]: (...args) => any;
+}
+
+class EventEmitter<Events extends IKeyFuncVal, Key extends keyof Events = keyof Events> {
+    private listeners: Map<Key, Set<Events[Key]>> = new Map();
+
+    emit<K extends Key>(eventName: K, ...restParams: Parameters<Events[K]>) {
```

Comment:
I have a bunch of ESLint warnings on methods and `REST_EMITTERS_CALLBACKS_PARAMS`. Could you please check if your IDE is using the right config? (`packages/cuba-rest-js/tslint.json`)